juntagrico/views/create_subscription.py

```diff
@@ -80,6 +90,88 @@ def select_start_date(request, signup_manager):
     return render(request, 'juntagrico/subscription/create/select_start_date.html', render_dict)
 
 
+class ExternalSignupForm(forms.Form):
+    '''
+    Defines the fields and validations for the external signup API, decouple from internal names
+    '''
+    first_name = forms.CharField()
+    family_name = forms.CharField()
+    street = forms.CharField()
+    house_number = forms.CharField()
+    postal_code = forms.CharField()
+    city = forms.CharField()
+    phone = forms.CharField()
+    email = forms.EmailField()
+    subscription_id = forms.ModelMultipleChoiceField(queryset=SubscriptionTypeDao.get_all_visible())
```

Comment:
DAOs will be deprecated. Use `SubscriptionType.objects.visible()` instead. Also below.

Reply:
Done